Car_Code/cheme_car_datalogging_codebase/cheme_car_datalogging_codebase.ino

```diff
@@ -376,6 +394,18 @@ void loop() // Loop (main loop)
   // // Update PID model
   // PID_loop();
 
+  if(ecValue >= EC_THRESH) {
+    // Stop driving
+    stop_driving();
+
+    // Indicate status to be finished
+    pixel.setPixelColor(0, 0, 0, 255);
+    pixel.show();
+
+    while (1)
+      ; // Do nothing for remainder of uptime
+  }
+  
   if (tempChange >= tempDiff)
```

Comment:
Also won't need this and it's accompanying code to calculate tempDiff & tempChange. Any references to those should be removed.